addons/mail/static/src/discuss/core/public_web/discuss_app/discuss_app_category_model.js

```diff
@@ -70,13 +68,13 @@ export class DiscussAppCategory extends Record {
 
     is_open = fields.Attr(true, { localStorage: true });
 
-    threads = fields.Many("mail.thread", {
+    channels = fields.Many("discuss.channel", {
         sort(t1, t2) {
             return this.sortThreads(t1, t2);
         },
         inverse: "discussAppCategory",
     });
-    threadsWithCounter = fields.Many("mail.thread", { inverse: "categoryAsThreadWithCounter" });
+    threadsWithCounter = fields.Many("discuss.channel", { inverse: "categoryAsThreadWithCounter" });
```

Comment:
Should rename field to `channelsWithCounter` and inverse `categoryAsChannelWithCounter`.